cmd/build.go

```diff
@@ -167,22 +167,24 @@ func setupBuildSingleTarget(ctx *context.Context) {
 	}
 }
 
-func setupBuildID(ctx *context.Context, id string) error {
+func setupBuildID(ctx *context.Context, ids []string) error {
 	if len(ctx.Config.Builds) < 2 {
 		log.Warn("single build in config, '--id' ignored")
 		return nil
 	}
 
 	var keep []config.Build
 	for _, build := range ctx.Config.Builds {
-		if build.ID == id {
-			keep = append(keep, build)
-			break
+		for _, id := range ids {
+			if build.ID == id {
+				keep = append(keep, build)
+				break
+			}
 		}
 	}
 
 	if len(keep) == 0 {
-		return fmt.Errorf("no builds with id '%s'", id)
+		return fmt.Errorf("no builds with ids '%s'", ids)
```

Comment:
could probably look more readable with something like:

```suggestion
		return fmt.Errorf("no builds with ids: %s", strings.Join(ids, ", "))
```